TUTORIAL-ASSETS.md

```diff
@@ -247,40 +245,33 @@ Hit continue and after a bit more processing you should now see your Assets insi
 
 ## Editing your Assets
 
-Now that you've imported your Assets, you can make any changes you wish to their position or orientation. To do this, you can either click on the model directly, or select from the Asset List panel by selecting the Asset Category.
-
-![Image showing the Asset Category List](images/cadtutorial/asset_list_category.png)
-
-Clicking on this will show you your list of Assets of the specified category. Here you can select them, which will move the camera and highlight the selected Asset, and also delete individual ones as required.
+Now that you've imported your Assets, you can make any changes you wish to their position or orientation. To do this, you can either click on the model directly, or select it from the Asset List panel. In the Asset List panel you can select your Assets, which will highlight the selected one, and also delete individual ones as required.
```

Comment:
Replace "In the Asset List panel you can select your Assets, which will highlight the selected one, and also delete individual ones as required." with "Selected Assets are highlighted onscreen.  In the Asset List panel you can also delete individual assets if required."